ITAQA/geography/converter.py

```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Utilities for geography-related conversions
+"""
+
+from ITAQA.geography.Italy import Province
+
+
+def lookup_province_name(province_enum):
```

Comment:
This method and its inverse are a bit verbose and require "double" maintenance.
I suggest generating a dictionary binding the province enums with the province names.
Reverse search can be performed at function call, or from a reverse dictionary generated at module import from the first one.

Reply:
Sounds absolutely reasonable! This was just a "quick implementation" with no real effort in it, made to complete the refactoring that was blocking other stuff, but of course it will be changed. Good proposal the dict with the reverse also!